src/custodia/server/args.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2015-2017  Custodia Project Contributors - see LICENSE file
```

Comment:
Can we separate the PR in at least two commits ?
One wehre you move around stuff to this new file without changing any code (aside from the code that mechanically needs to change to support loading stuff from the other file).
And then a commit on top that makes the instances changes.